src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -35,6 +36,7 @@
     private final String clientId;
     private final String clientSecret;
     private final String baseUrl;
```

Comment:
Not really, was used by the `AuthorizeUrlBuilder` but now that I see we can replace that parameter with the `HttpUrl`. Let me fix it. 👍